tests/api/test_categories.py

```diff
@@ -4,7 +4,7 @@
 
 from app import app, db
 from models import Category
-from tests.test_utils import clearDatabase
+from tests.test_utils import clearDatabase, createCategory, auth_headers, require_auth0_secrets
```

Comment:
See issue #56 -- at some point I should go back and clean up the styles in this codebase because I am not an experienced enough Python programmer to name methods consistently 😅